app/views/ThemeView.tsx

```diff
@@ -58,79 +57,112 @@ interface ITheme {
 	group: string;
 }
 
-type IThemeViewProps = IBaseScreen<SettingsStackParamList, 'ThemeView'>;
+const Item = ({
```

Comment:
We discussed this on Today's daily.
One component per file, just to make it easier to read and files with fewer lines :)

Reply:
I asked to @GleidsonDaniel and he told me that you both agreed to keep the item in the same file. So, considering that, I can approve this PR.